### DISCUSSION TITLE
I get that I can generate styles on the fly, but how to I pre-render them for a static site?

### DISCUSSION THREAD

**Discussion by panphora** [master-co/css#33] [Help]
I'm working with plain HTML & CSS.

I included the CDN scripts:

```
    <script src="https://unpkg.com/@master/style"></script>
    <script src="https://unpkg.com/@master/styles"></script>
```

This gives me live style generation, which is great!

But is there a way to remove these CDN scripts and generate the styles for an HTML page as a CSS file — so I can ship it in one static bundle?

**Comment by 1aron** [comment]
We're currently undergoing a comprehensive revision and rectification of the official website. Now you can import a package of `@master/css`.

```html
<script src="https://unpkg.com/@master/css"></script>
```

We've provide hybrid rendering where you can pre-generate CSS (SSR) from the server.
If you want to integrate with build tools at compile time, it's not currently available, but we will provide this part in the future to make it more comprehensive.
